src/main/java/bio/terra/cli/command/workspace/Create.java

```diff
@@ -28,13 +28,21 @@ public class Create extends WsmBaseCommand {
   // Variable is `id` instead of `userFacingId` because user sees it with `terra workspace create`
   private String id;
 
+  @CommandLine.Option(
+      names = "--platform",
+      description = "Set the Cloud platform: ${COMPLETION-CANDIDATES}.",
+      defaultValue = "GCP")
```

Comment:
I noticed we weren't showing default value with `terra workspace create`. Looks like you need `showDefaultValue`:

```
  @CommandLine.Option(
      names = "--platform",
      description = "Set the Cloud platform: ${COMPLETION-CANDIDATES}.",
      showDefaultValue = CommandLine.Help.Visibility.ALWAYS,
      defaultValue = "GCP")
  private CloudPlatform cloudPlatform;
```

Reply:
Yes, noted and added.